bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/model/UserEvaluations.kt

```diff
@@ -6,4 +6,7 @@ import com.squareup.moshi.JsonClass
 data class UserEvaluations(
   val id: String,
   val evaluations: List<Evaluation> = emptyList(),
+  val createdAt: String,
+  val forceUpdate: Boolean,
+  val archivedFeatureIds: List<String> = emptyList(),
```

Comment:
Fields with default value should come last.

```suggestion
  val id: String,
  val createdAt: String,
  val forceUpdate: Boolean,
  val evaluations: List<Evaluation> = emptyList(),
  val archivedFeatureIds: List<String> = emptyList(),
```

Reply:
Thank you for reviewing.

I applied your suggestion in https://github.com/bucketeer-io/android-client-sdk/pull/88/commits/ce54007dd0f2ff206001e38047b8eefa1b3f53ef

Please help me for take a look again